www/modules/recalls/services/RecallSearchService.js

```diff
@@ -0,0 +1,119 @@
+var main = require('../main'),
```

Comment:
Lets move this service to a dedicated module called "openfda". That module should contain a constant that allows for specifying the server endpoint and api key values. The module should then expose this service as "FoodEnforcement".